packages/blade/src/components/BottomSheet/BottomSheet.native.tsx

```diff
@@ -236,6 +237,17 @@ const _BottomSheet = ({
       <DropdownContext.Provider value={dropdownProps}>
         <BottomSheetContext.Provider value={contextValue}>
           <GorhomBottomSheet
+            style={{
+              shadowColor: 'hsla(217,56%,17%,0.64)',
+              shadowOffset: {
+                width: 0,
+                height: -18,
+              },
+              shadowOpacity: 0.2,
+              shadowRadius: 12,
```

Comment:
Let's add a comment on what this shadow is suppose to be close to, lowRaised, midRaised, or highRaised.